README.md

```diff
@@ -6,5 +9,141 @@ Exclusive Fairness Criteria in Classification Problems](https://arxiv.org/abs/22
+```bash
+faim-experiment --prepare-data DATASET
+```
+where `DATASET` is one of:
+* `synthetic-from-paper`
+* `compas`
+* `zalando` [waiting for permission to release, contact us for more information]
+
+The dataset will be downloaded, and prepared to a folder called `prepared-data`.
+
+The following sections include info about each dataset:
+
+###### synthetic data
+The raw dataset in the GitHub repo corresponds to synthetic prediction and ground truth scores for two groups,
+for each group sampling from a corresponding binormal distribution.  This data can be considered to fall under the
+umbrella of FAIM's [LICENSE](LICENSE).
+
+###### compas
+The raw data was obtained **[here]** with [this] license.
+
+###### zalando
+**Under construction, more information to follow!**
+
+#### Run Experiment
+
+Having prepared data following the instruction above, you are ready to run a FAIM experiment:
+```bash
+faim-experiment --run PREPARED-DATASET LOW_SCORE_VAL,HIGH_SCORE_VAL THETAS PREPARED_DATA_FILEPATH
+```
+
+`PREPARED-DATASET` is now one of the following options (depending on what has been prepared):
+* `syntheticTwoGroups` (prepared using `--prepare-data synthetic*`)
+* `compasGender` (prepared using `--prepare-data compas`)
+* `compasRace` (prepared using `--prepare-data compas`)
+* `compasAge` (prepared using `--prepare-data compas`)
+* `zalando` (prepared using `--prepare-data zalando`) [waiting for permission to release, contact us for more information]
+
+`LOW_SCORE_VAL,HIGH_SCORE_VAL` are two numbers that define the score range.
+
+`THETAS` correspond to the fairness compremise you want. There are three thetas per group corresponding to the
+desired amount of the three fairness criteria that the system should achieve:
+1. group calibration
+2. equalized false negative rates
+3. equalized false positive rates
+
+Note, as discussed in the paper, thetas = 1,1,1 does not indicate that the system will simultaneously achieve all
+three (mutually incompatible) fairness criteria, but rather the result will be a compromise between all three.
+
+See the [paper](https://arxiv.org/abs/2212.00469) for more details.
+
+Finally, `PREPARED_DATA_FILEPATH` corresponds to the filepath of the prepared data.
+
+###### Examples
+Run all of the following from the same folder where `faim-experiment --prepare-data` was run.
+
+In each example, a FAIM post-processor is trained and evaluated with results saved under the `results` folder:
+* Train FAIM model on synthetic dataset without any fairness correction
+  ```bash
+  faim-experiment --run syntheticTwoGroups 0.1 0,0,0,0,0,0 prepared-data/synthetic/2groups/2022-01-12/dataset.csv
+  ```
+* Train FAIM model on synthetic dataset to achieve a combination of all three fairness criteria.
+  ```bash
+  faim-experiment --run syntheticTwoGroups 0.1 1,1,1,1,1,1 prepared-data/synthetic/2groups/2022-01-12/dataset.csv
+  ```
+
+#### Visualize Results
+**Needs documentation!**
+
```

Comment:
plots are done during the execution of faim (i.e., during training) and saved into folder results/DATASET-NAME/

Reply:
I couldn't get it to work.  Could you write the section and commit here and then I'll see if the instructions work for me with some expected results?